Algorithm.CSharp/CustomDataIndicatorExtensionsAlgorithm.cs

```diff
@@ -0,0 +1,95 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using QuantConnect.Data;
+using QuantConnect.Data.Custom;
+using QuantConnect.Data.Market;
+using QuantConnect.Indicators;
+
+namespace QuantConnect.Algorithm.CSharp
+{
+    /// <summary>
+    /// The algorithm creates new indicator value with the existing indicator method by Indicator Extensions
+    /// Demonstration of using the external custom datasource Quandl to request the VIX and VXV daily data
+    /// </summary>
+    /// <meta name="tag" content="using data" />
+    /// <meta name="tag" content="using quantconnect" />
+    /// <meta name="tag" content="custom data" />
+    /// <meta name="tag" content="indicators" />
+    /// <meta name="tag" content="indicator classes" />
+    /// <meta name="tag" content="plotting indicators" />
+    /// <meta name="tag" content="charting" />
+    public class CustomDataIndicatorExtensionsAlgorithm : QCAlgorithm
+    {
+        private const string VIX = "CBOE/VIX";
+        private const string VXV = "CBOE/VXV";
+        private SimpleMovingAverage smaVIX;
+        private SimpleMovingAverage smaVXV;
+        private IndicatorBase<IndicatorDataPoint> ratio;
```

Comment:
Minor comment:
In C#, we use the underscore pattern for private global variables: 
```csharp
private const string _vix = "CBOE/VIX";
private const string _vxv = "CBOE/VXV";
private SimpleMovingAverage _smaVIX;
private SimpleMovingAverage _smaVXV;
private IndicatorBase<IndicatorDataPoint> _ratio;
```

Reply:
I suppose if we're trying to make a point or follow a convention while calling these `private` then we should do the same with the Python code and use the PEP 8 `_` or `__` prefix for the same reason.